packages/perseus-core/src/widgets/matcher/matcher-util.test.ts

```diff
@@ -0,0 +1,41 @@
+import getMatcherPublicWidgetOptions from "./matcher-util";
+
+import type {PerseusMatcherWidgetOptions} from "../../data-schema";
+
+describe("getMatcherPublicWidgetOptions", () => {
+    it("should return a shuffled right array when order doesn't matter to remove the correct order information", () => {
+        // Arrange
+        const options: PerseusMatcherWidgetOptions = {
+            labels: ["**Claims**", "**Evidence**"],
+            padding: true,
+            orderMatters: false,
+            left: ["Fuel", "Plate", "Hydrogen", "Average", "Billion"],
+            right: ["Stars", "Earth", "Life", "Rapid", "Milky Way"],
+        };
+
+        // Act
+        const publicWidgetOptions = getMatcherPublicWidgetOptions(options);
+
+        // Assert
+        expect(publicWidgetOptions.right).not.toEqual(options.right);
+        expect(publicWidgetOptions.left).toEqual(options.left);
```

Comment:
Now that we're using Math.random(), we can't expect a specific outcome because the shuffle should be different each time. To achieve this, I did have to multiply the random output by 100 though.

Reply:
Will update the Math.random() call to just Math.random, which will still allow the tests here to pass